build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "org.springframework.boot" version "3.4.1"
+    id "org.springframework.boot" version "3.4.2"
```

Comment:
the module `org.springframework.session:spring-session-core` is still on version 3.4.1

Reply:
Seems 3.4.2 doesnt exist yet, ill keep PR open for now